conda/core/solve.py

```diff
@@ -2153,6 +2155,155 @@ def working_state_reset(self):
         self.final_environment_specs = None
 
 
+class TrackedMap(MutableMapping):
+    def __init__(self, name, data=None):
+        self._data = data
+        self._name = name
+        self._logger = logging.getLogger(self.__class__.__name__)
```

Comment:
I think we want to use a logger name here that includes the full import path to be able to namespace it correctly.